modules/tinymce/src/themes/silver/test/ts/browser/editor/ToolbarPersistTest.ts

```diff
@@ -0,0 +1,57 @@
+import { Chain, Log, Pipeline } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { Editor as McEditor, UiChains } from '@ephox/mcagar';
+import { SugarElement, SugarBody, Insert, Focus, Remove, Visibility } from '@ephox/sugar';
+import { Fun } from '@ephox/katamari';
+import Theme from 'tinymce/themes/silver/Theme';
+import Editor from 'tinymce/core/api/Editor';
+
+UnitTest.asynctest('Toolbar persist test', (success, failure) => {
```

Comment:
I know we've been pretty bad with naming tests in silver, but maybe update this to follow the TinyMCE naming convention used everywhere else? eg `browser.tinymce.themes.silver.editor.ToolbarPersistTest`